client/sections-preloaders.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Internal dependencies
+ */
+import { preload } from './sections-helper';
+
+export function preloadEditor() {
+	preload( 'gutenberg-editor' );
+}
```

Comment:
Should this function also include `preload( 'post-editor' )`?

```suggestion
export function preloadEditor() {
	preload( 'gutenberg-editor' );
	preload( 'post-editor' );
}
```

Reply:
Uh, sorry @saramarcondes - this is expected.

It was changed because of this review request: https://github.com/Automattic/wp-calypso/pull/45934#discussion_r495773143 but I forgot to update the PR description.

Could you take another look?